src/fill_ontologies.py

```diff
@@ -13,200 +14,318 @@
+    DB restrictions
     e. Query the OLS/HCAO with database restrictions
-    f. Return ontologies that match the search and ask the user which one is correct.
-    g. Save spreadsheet with the same name + _ontologies
+    f. If array delimiter `||` is detected in the cell, each term is automatically searched and the returned curations
+      are concatenated in the spreadsheet.
+    g. If ZOOMA option enabled, query the ZOOMA API for any available curation
+    h. If `--keep` option specified, no existing ontologies will be overwritten.
+    i. Return ontologies that match the search and ask the user which one is correct.
+    j. Save spreadsheet with the same name + _ontologies
 
 1. Instruction of use
 
     **Install requirements** first (See above)
 
     ```
     cd src/
-    python3 fill_ontologies.py -s <path_to_spreadsheet>
+    python3 fill_ontologies.py -s <path_to_spreadsheet> [OPTIONS -z -k]
     ```
     The script takes some time to get started (It's loading a SchemaTemplate object). After approx 30 secs/1 min, it
     will begin the search, with the following possible outputs:
-    - `Found exact match for term 'x' (Ontology: 'y')`: The program has found an ontology that matches perfectly the spreadsheet value
-    - `Ontology with label 'x' ('y') has been found (<spreadsheet text value>). Is this the term that you were looking for? [Y/n/m/multi]:
-        - Input 'Y', 'y', 'yes', 'Yes', '': The tool will fill the spreadsheet with that ontology.
+    - `Found exact match for term 'x' (Ontology: 'y')`: The program has found an ontology that matches perfectly the
+    spreadsheet value
+    - `Found high confidence, HCA match for term 'x' (Ontology: y)`
+    - `x matches found for your search term 'y' for field <spreadsheet text value>.`
+        - Numbered list of matches in format (ZOOMA info show with option -z):
+            - n. ontology_label - ontology_id. ZOOMA: confidence: (HIGH/GOOD/MEDIUM/LOW), source: <SOURCE>.
+            - Enter the appropriate number or 'm' for manual input or 'none' to skip this term
+        - Input n: The tool will fill the spreadsheet with that ontology.
         - Input 'm': You'll be prompted to introduce the term that you want to look for
-        - Input 'multi': You'll be prompted to introduce the terms, one by one, and then introduce break.
-        - Any other input: Next term in the list of ontologies found for the term
-    - No ontology was found for this term. Please input it manually: Input the term that you want to search. Can also type "none".
+        - Input 'none': The ontology cells will be filled with empty strings
+    - No ontology was found for this term. Please input it manually: Input the term that you want to search.
 """
 
-import requests as re
+import requests as rq
 import argparse
 import openpyxl
 from ingest.template.schema_template import SchemaTemplate
 from openpyxl.utils import get_column_letter
+import pickle
+import re
+import os
+import sys
+
```

Comment:
```suggestion
import os
import pickle
import requests as rq
import re

import argparse
import openpyxl
from openpyxl.utils import get_column_letter

from ingest.template.schema_template import SchemaTemplate

```

Just a small touch, I'm trying to follow pep8 more often! https://www.python.org/dev/peps/pep-0008/#imports

Reply:
cool thanks!